pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -82,6 +102,8 @@ func (ds *dockerService) RunPodSandbox(config *runtimeapi.PodSandboxConfig) (str
 		return "", fmt.Errorf("failed to create a sandbox for pod %q: %v", config.Metadata.Name, err)
 	}
 
+	ds.setNetworkReady(createResp.ID, false)
```

Comment:
Yes, still a small race.  But IMHO SetUpPod is what takes the longest time, and that's what was causing the race.  We can't really address that unless we can use name/namespace as the networkReady key, and we can't do that unless we can add that to the checkpoint.

One option I toyed with was adding docker container labels with the name/namespace, which we could get from ListContainers().  What do you think?  We already use labels int he Checkpoint code, and the rkt runtime does something similar.  However, I'd like to do that as a follow-up PR if that seems workable...